terraform/terraform-modules/ecs/container-definitions/pds-validate-containers.json

```diff
@@ -0,0 +1,21 @@
+[
+   {
+       "name": "pds-validate-task",
+       "image": "${pds_validate_ecr_image_path}",
+       "cpu": 0,
+       "memory": 128,
+       "portMappings": [],
+       "essential": true,
+       "environment": [],
+       "mountPoints": [],
+       "volumesFrom": [],
+       "logConfiguration": {
+           "logDriver": "awslogs",
+           "options": {
+               "awslogs-group": "/ecs/pds-validate",
+               "awslogs-region": "us-west-2",
```

Comment:
This seems like something we may want to change at some point, but not a huge deal...

Reply:
@ramesh-maddegoda can these be passed in as env variables during workflow execution?

Reply:
@sjoshi-jpl and @jordanpadams , I externalized these values as variables that we pass in a tfvars file (same approach we used for Nucleus)

Reply:
Looks good now. Approved!